lapack/testlapack/dgesvd.go

```diff
@@ -178,7 +178,7 @@ func svdCheckPartial(t *testing.T, impl Dgesvder, job lapack.SVDJob, errStr stri
 	if shortWork {
```

Comment:
Note for this file that there are ineffasigns to `jobU` and `jobV` that I don't understand how to resolve. @btracey do you have suggestions?

Reply:
Please wait with this, I've completely rewritten the Dgesvd test which I will submit soon.

Reply:
I'll drop this commit.